limit-modified-date.php

```diff
@@ -11,22 +11,47 @@
 
 class Limit_Modified_Date {
 
+	/**
+	 * Limit Modified Date enabled post meta key.
+	 *
+	 * @var string
+	 */
 	private $meta_key = 'limit_modified_date';
 
-	private $version = '1.0';
+	/**
+	 * Limit Modified Date modified date post meta key.
+	 *
+	 * @var string
+	 */
+	private $last_mod_meta_key = 'last_modified_date';
+
+	/**
+	 * Limit Modified Date nonce key.
+	 *
+	 * @var string
+	 */
+	private $nonce_key;
+
+	/**
+	 * Limit Modified Date asset version.
+	 *
+	 * @var string
+	 */
+	private $asset_version = '1.0';
 
-	function __construct() {
+	public function __construct() {
+		$this->nonce_key = $this->meta_key . '_nonce';
 
 		// Use original modified date
-		add_action('wp_insert_post_data', array( $this, 'use_original_modified_date' ), 20, 2 );
+		add_action( 'wp_insert_post_data', [ $this, 'use_original_modified_date' ], 20, 2 );
```

Comment:
@billerickson One minor question that came up during code review - is there a specific reason why this is set as priority `20`? If so maybe we can document it here

Reply:
I wish I documented it at the time. I don't recall the reason, but I changed it from 10 to 20 so I think there's a reason. https://github.com/billerickson/Limit-Modified-Date/commit/71a356dcec391d7a5bcc558038cd690d0fe9955b